vscode-extension/src/extension.ts

```diff
@@ -1,27 +1,290 @@
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
-import * as vscode from 'vscode';
+import { workspace, window, commands, extensions, Uri, ExtensionContext } from 'vscode';
+import * as process from 'process';
+import * as cp from 'child_process';
+import * as util from 'util';
+import { posix } from 'path';
 
 // This method is called when your extension is activated
 // Your extension is activated the very first time the command is executed
-export function activate(context: vscode.ExtensionContext) {
-
-	// Use the console to output diagnostic information (console.log) and errors (console.error)
+export function activate(context: ExtensionContext) {
 	// This line of code will only be executed once when your extension is activated
-	vscode.window.onDidOpenTerminal(async (event) => {
-		runDevboxShell();
+	initialCheckDevboxJSON();
+	// Creating file watchers to watch for events on devbox.json
+	const fswatcher = workspace.createFileSystemWatcher("**/devbox.json", false, false, false);
+	fswatcher.onDidDelete(e => {
```

Comment:
Don't need the curly brakets?

Reply:
fair point 😄  updated
I used to work with a senior JS dev who enforced the use if curly brackets at all times.